permits/models.py

```diff
@@ -67,11 +113,11 @@ class Meta:
         verbose_name_plural = _("contacts à saisir")
 
     def __str__(self):
-        return str(self.type) + '(' + str(self.works_type) + ')'
+         return self.get_type_display() + ' (' + str(self.works_type) + ')'
```

Comment:
Il y a un espace en trop au début de la ligne:

```suggestion
        return self.get_type_display() + ' (' + str(self.works_type) + ')'
```

Reply:
Done